src/test/scala/com/vngrs/etl/exceptions/NoSchemaExceptionSpec.scala

```diff
@@ -0,0 +1,23 @@
+package com.vngrs.etl.exceptions
+
+import com.vngrs.etl.BaseSpec
+
+/**
+  * No Schema Exception Specs
+  */
+// Following wart does not work with scalatest's intercept functionality
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+class NoSchemaExceptionSpec extends BaseSpec {
```

Comment:
There are better ways https://github.com/scoverage/sbt-scoverage#exclude-classes-and-packages